aiomcache/helpers.py

```diff
@@ -0,0 +1,42 @@
+import asyncio
+from enum import IntEnum
+import pickle
+
+
+# https://github.com/lericson/pylibmc/blob/master/src/_pylibmcmodule.h#L63
+class PyLibMCFlags(IntEnum):
```

Comment:
The comment where these are defined says they come from python-memcached, which we already depend on. So, shouldn't we just be able to just import all these anyway?

Reply:
unfortunately they're private: https://github.com/linsomniac/python-memcached/blob/master/memcache.py#L133-L137

Reply:
Hmm, if memcached is already processing these flags in the client, why do we need to do it again here?

Forgive me if that's a stupid question, I'm not too familiar with the code or how these flags work.

Reply:
you know I've already completely forgotten, let me investigate

Reply:
so this client is a raw asyncio client, it doesn't use memcached.  What it's doing is duplicating what that client does.  memcached is used for testing, to ensure aiomemcache can read data written by it and vice versa.

Reply:
Ah, yes, my mistake.